groups/sig-cluster-lifecycle/groups.yaml

```
@@ -480,6 +480,22 @@ groups:
       - dlipovetsky@d2iq.com
       - neolit123@gmail.com
 
+  - email-id: sig-cluster-lifecycle-kubespray-alerts@kubernetes.io
+    name: sig-cluster-lifecycle-kubespray-alerts
+    description: |-
+      Group for Kubespray maintainers to receive CI system notifications.
+    settings:
+      WhoCanDiscoverGroup: "ANYONE_CAN_DISCOVER"
+      WhoCanViewGroup: "ANYONE_CAN_VIEW"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      WhoCanJoin: "ANYONE_CAN_JOIN"
+      MessageModerationLevel: "MODERATE_ALL_MESSAGES"
+      ReconcileMembers: "false"
```

Should those be :
```suggestion
      WhoCanJoin: "INVITED_CAN_JOIN" 
      MessageModerationLevel: "MODERATE_NONE" 
      ReconcileMembers: "true"
```
So the list stay manageable from this file and alets come through ?

---

there was something tricky related to allowing bots to post on the ML.
the params worked for the kubeadm-alerts ML, but feel free to experiment.

---

IMO having a file in git to handle membership is cleaner and less opaque.
For the reconcile bit, if it handles the owners I think it does not matter much ; owners and members don't make a meaningful distinction in that case, IMHO.

---

Good idea. It has been changed.